Tracking/src/ClicEfficiencyCalculator.cc

```diff
@@ -498,11 +500,15 @@ void ClicEfficiencyCalculator::processEvent( LCEvent* evt ) {
           TLorentzVector particleVector2;
           particleVector2.SetPxPyPzE(particle2->getMomentum()[0],particle2->getMomentum()[1],particle2->getMomentum()[2],particle2->getEnergy());
           double DR = particleVector.DeltaR(particleVector2);
+          distancesR.push_back(DR);
```

Comment:
If you don't need the DR's later on, you can just do this here
```c++
minDR = std::min(DR, minDR);
```
and define minDR before the loop , initialized to some large value (DBL_MAX)

Reply:
This will also work if there is just one particle